modules/model/shared/src/main/scala/gem/Parallax.scala

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2016-2020 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gem
+
+import cats.Order
+import cats.implicits._
+
+/**
+  * Parallax in mas
+  * Store the value in mas to avoid rounding errors
+  */
+case class Parallax(mas: BigDecimal) extends Serializable
```

Comment:
This is an angle but in practice it is always a small number and it can be negative thus it maybe better stored as `BigDecimal`

Reply:
The code looks good but in general though, we have stored this kind of thing as an `Angle`.  For example, `Offset` components are typically thought of as signed arcseconds and we keep them as `Angle` with optics to extract signed decimal arcseconds.  My inclination is that it's useful to store everything as an `Angle` because otherwise, with a collection of `BigDecimal`, it isn't clear how they combine and relate to one another.

Concretely, I think this is along the lines of what I would suggest:

```scala
final case class Parallax(toAngle: Angle)

object Parallax extends ParallaxOptics { 
...
}

sealed trait ParallaxOptics { this: Parallax.type =>

  lazy val angle: Iso[Parallax, Angle] =
    Iso(_.toAngle, Parallax.apply)

  /**
   * This `Parallax` as signed decimal milliarcseconds.
   */
  lazy val signedMilliarcseconds: SplitMono[Parallax, BigDecimal] =
    SplitMono.fromIso(angle).composeSplitMono(Angle.signedDecimalMilliarcseconds)

}
```

Reply:
My first approach was to use `Angle`, my worried was that `Parallax` is generally in micro arcseconds and perhaps that would be an issue?

Reply:
It seems the smallest parallax you can measure is around 10 micro arcseconds. Given that Angle stores values as `sealed class Angle protected (val toMicroarcseconds: Long)` I guess it should be ok to use `Angle`

Reply:
That's a good question and I'm not 100% sure.  In the `ocs` codebase (in the OT specifically) we show parallax in _milli_ arcseconds (mas) which is 1000X the resolution of the smallest `Angle` we can represent.  A true sub-microarcsecond parallax wouldn't contribute much to proper motion, I would think.

Reply:
[A note from LCO](https://lco.global/spacebook/distance/parallax-and-distance-measurement/) seems to imply that measuring parallax at sub-millisecond resolution is difficult even for space-based telescopes.  If so, `Angle` would be fine.